README.md

```diff
@@ -1 +1,30 @@
-Feedhenry Sample Service Using Hard Coded Data for Datasources
+# Feedhenry Sample Service Using Hard Coded Data for Datasources
+
+This is a sample RESTful service to load a list of **months** that is properly formatted as a Data Source which can be used in Apps created with Form Builder.
```

Comment:
Would probably have:

This is a sample RESTful service to load a list of **months** that is properly formatted for a Data Source.